packages/rpc/src/methods/send-transfer.ts

```diff
@@ -6,11 +6,29 @@ export interface SendTransferRequestParams extends RpcParameterByName {
   amount: string;
 }
 
+export interface RpcSendTransferParamsLegacy extends SendTransferRequestParams {
+  network: string;
+}
+
+interface TransferRecipientParam {
+  address: string;
+  amount: string;
+}
+
+export interface RpcSendTransferParams extends RpcParameterByName {
+  account?: number;
+  recipients: TransferRecipientParam[];
+  network: string;
+}
```

Comment:
Can all these new types be describes as zod schemas, and the types inferred with `z.infer`?

Reply:
Yeah, I will conform to the patterns elsewhere. 

What is the advantage of just using zod for types though? I guess down the line validators can be exported?

Reply:
Would you be happy with a fast follow on the zod stuff? Would ideally like to keep this scope down - it should be 1:1 replacements but I feel is different in scope than this effort, which is trying to move `extension` rpc types to our package.